Kitodo/src/main/java/org/kitodo/forms/TemplateForm.java

```diff
@@ -192,6 +201,15 @@ public String save() {
                 return null;
             }
 
+            try {
+                if (this.template.getTasks().isEmpty()) {
+                    Reader reader = new Reader(this.template.getWorkflow().getFileName());
+                    this.template = reader.convertWorkflowToTemplate(this.template);
+                }
+            } catch (IOException e) {
+                logger.error("Problem with diagram");
```

Comment:
Can you please provide a more meaningful message as this message did not help to locate the real error reason. Maybe you should even inform the user with error.